### PR TITLE
 feat: use nps for scripts

### PR DESCRIPTION
**Pull request by jyash97**
In testing I was getting some versioning issues and all the packages are currently updated. But I checked with pwd to ensure we are in the right directory.

### CODE REVIEW

File: scripts/build-web-example.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+echo Upgrading examples RS version to $1
+
+var=`pwd`
+for d in packages/web/examples/*/ ; do
+    cd $var/$d
+    yarn add @appbaseio/reactivesearch@$1
```
Comment (bietkul) [inline scripts/build-web-example.sh]: Why do we need to use yarn here, again it'll take a long time to execute this script?
Can't you just modify the package.json with the particular version?

**Review by bietkul — CHANGES_REQUESTED** [review]

File: scripts/build-web-example.sh (hunk 1 above)
Comment (jyash97) [reply scripts/build-web-example.sh]: have removed `yarn` & now we are just changing package version.

File: scripts/update-web-workspace.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+echo Upgrading related Workspace to $1
+
+cd packages/playground && yarn add @appbaseio/reactivesearch@$1 && cd ../maps && yarn add @appbaseio/reactivesearch@$1 && cd ../..
```
Comment (bietkul) [inline scripts/update-web-workspace.sh]: remove `yarn` from here too.
and you're not updating the demos inside `site` folder.
I think only one script can work with a recursive function
- Start from the root and traverse all the folder
- Check if the folder has package.json with reactivesearch dependency
- update it to the latest
- move forward and repeat it for all the nested folders until the whole directory is covered

**Review by bietkul — CHANGES_REQUESTED** [review]

File: scripts/update-web-workspace.sh (hunk 2 above)
Comment (jyash97) [reply scripts/update-web-workspace.sh]: > and you're not updating the demos inside site folder.

For `site` we already have scripts & we need yarn there to update `yarn.lock` & create a build.

> remove yarn from here too.

We have yarn.lock file in playground as well. So we may need to use yarn their also.

**Review by lakhansamani — APPROVED** [review]

**Review by bietkul — APPROVED** [review]

**Comment by bietkul** [comment]
@jyash97 Can you resolve the conflicts?

**Comment by jyash97** [comment]
Done 🏁

**Pull request merged**